### ISSUE TITLE
Editor: Visually display the local user's current focus

### ISSUE THREAD

**Issue by EricWittmann** [Apicurio/apicurio-studio#621]
We need to evaluate **all** of the UI elements we're using currently in the editor and make sure we have sensible keyboard navigation and shortcuts for them.  Tab should move the user's focus from element to element, and appropriate keyboard controls should be available for each control.
